config.schema.yaml

```diff
@@ -40,8 +40,8 @@ properties:
   reference_variants:
     type: string
     description: Path to reference variants, if not provided then variant will be called by freebayes
-    default:
+    default: ""
   phasing_ground_truth:
     type: string
     description: Path to phased variants used to compute phasing stats.
-    default: .
\ No newline at end of file
+    default: ""
```

Comment:
There’s a missing newline at the end of the file

Reply:
Fixed in new commit.